seatunnel-formats/seatunnel-format-json/src/main/java/org/apache/seatunnel/format/json/debezium/DebeziumJsonDeserializationSchema.java

```diff
@@ -93,8 +96,7 @@ public void deserialize(byte[] message, Collector<SeaTunnelRow> out) throws IOEx
             } else if (OP_UPDATE.equals(op)) {
                 SeaTunnelRow before = convertJsonNode(payload.get("before"));
                 if (before == null) {
-                    throw new SeaTunnelJsonFormatException(
-                            CommonErrorCodeDeprecated.UNSUPPORTED_DATA_TYPE,
+                    throw new IllegalStateException(
```

Comment:
why changed this?

Reply:
https://github.com/apache/seatunnel/blob/87db4d1f8bfc6273a727d5ed77e642a4fbee5e72/seatunnel-common/src/main/java/org/apache/seatunnel/common/exception/CommonErrorCodeDeprecated.java#L34

https://github.com/apache/seatunnel/blob/87db4d1f8bfc6273a727d5ed77e642a4fbee5e72/seatunnel-common/src/main/java/org/apache/seatunnel/common/exception/CommonErrorCode.java#L24-L25

I think this code block aims to verify if payload meet the specification of Debezium. It appears to be unrelated to 
seatunnel data types and `UNSUPPORTED_DATA_TYPE`.